pkgs/misc/themes/yaru/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, meson, ninja, sassc, gtk3, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "yaru-theme-${version}";
+  version = "18.10.7";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu";
+    repo = "yaru";
+    rev = version;
+    sha256 = "1aryri3jk16yrgrhr6c16lfci16cknra28h4k02kfxb51xx7ih60";
+  };
+
+  nativeBuildInputs = [ meson ninja sassc python3 ];
+  buildInputs = [ gtk3 ];
```

```suggestion
  buildInputs = [ gtk3 gnome3.gnome-themes-extra ];
```
I think so because it's in [debian control](https://github.com/ubuntu/yaru/blob/b028423826bf2f2867fba1866f1cb170d4f817af/debian/control#L30)